install/kubernetes/helm/subcharts/gateways/values.yaml

```diff
@@ -9,6 +9,12 @@ enabled: true
 
 istio-ingressgateway:
   enabled: true
+  #
+  # ingress sds configuration
```

Comment:
Can you add more comments - users don't know or care what 'sds' means, it's an implementation detail. 

AFAIK we'll use values.yaml to generate docs - treat it as highly user visible.